NEW_README.md

```diff
@@ -188,29 +188,43 @@ bundle exec rake opsworks_performance    # (takes about 1.5 hours)
 
 ## Soak performance tests
 
-In order to execute a Soak performance run:
+The soak performance test executes a long-running scenario under medium load:
+* 14 days
+* 600 agents
+* `role::by_size::large`
 
-* Follow the instructions from
-[Just set up PE and Gatling](#set-up-puppet-enterprise-and-gatling-but-do-not-execute-a-gatling-scenario).
+### Provision, tune, run
```

Comment:
This header doesn't make sense any more.  It can probably just go entirely

Reply:
Removed